tests/test_process.py

```diff
@@ -221,6 +221,21 @@ def test_process_logs(self):
             self.assertIsNotNone(log.lvl)
             self.assertIsNotNone(log.msg)
 
+    def test_start_new_session(self):
+        # GIVEN
+        self.given_ngrok_installed(self.pyngrok_config.ngrok_path)
+
+        # WHEN
+        config = self.pyngrok_config
+        config.start_new_session = True
+        ngrok_process = process._start_process(self.pyngrok_config)
+
+        # THEN
```

Comment:
i wanted to test that the log contained the warning if sys.version_info.major < 3, but didn't see a low profile way to do it.